quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/transaction/QuarkusJtaTransactionManagerLookup.java

```diff
@@ -70,6 +69,6 @@ public int order() {
 
     @Override
     public boolean isSupported() {
-        return !Profile.isFeatureEnabled(Profile.Feature.MAP_STORAGE);
+        return true;
```

Comment:
I'm just wondering if it makes sense to implement `EnvironmentDependentProviderFactory` after all when it becomes supported always.

Reply:
Thanks for noting this, I forgot about that part. In the latest commit this has now been cleaned up.